packages/astro/src/@types/typed-emitter.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * The MIT License (MIT)
+ * Copyright (c) 2018 Andy Wermke
+ * https://github.com/andywer/typed-emitter/blob/9a139b6fa0ec6b0db6141b5b756b784e4f7ef4e4/LICENSE
```

Comment:
Note that I brought this package internally because when Vite crawls through package.jsons, this one has a main but the main doesn't export anything so Vite warns about it not being a real ESM module. This is only a (small) types package, so brought it in.